### DISCUSSION TITLE
Where do we get the last documentation?

### DISCUSSION THREAD

**Discussion by melMass** [web-infra-dev/rspack#7779] [Q&A] [answered]
Hi,

I just started using rspack and all the things I've looked for are outdated in the doc, namely:
- setting the dev port is referenced as the `devServer` in the doc, now it's just `server`?
- rspack plugins must be added to `config.tools.rspack` and not `config.plugins`

Is there a way to get the latest doc somwhere? 

Thanks

**Comment by hardfist** [comment, marked as answer]
maybe you mix rspack and rsbuild docs？ can you paste the link that you think cintains wrong content

**Reply by melMass** [reply]
I 100% did yep my bad (I'm actually using rsbuild)
